source/guides/rapids-docker-with-cuda.md

```diff
@@ -0,0 +1,369 @@
+# Building RAPIDS containers from a custom base image
+
+This guide provides instructions to add RAPIDS and CUDA to your existing Docker images. This approach allows you to integrate RAPIDS libraries into containers that must start from a specific base image, such as application-specific containers.
+
+The CUDA installation steps are sourced from the official NVIDIA CUDA container images maintained at the [NVIDIA CUDA Container Images GitLab repository](https://gitlab.com/nvidia/container-images/cuda).
+
+```{docref} ../custom-docker
+If you have the flexibility to choose your base image, see the Custom RAPIDS Docker Guide which starts from NVIDIA's official CUDA base images for a simpler setup.
+```
+
+## Overview
+
+If you cannot use NVIDIA's CUDA container images, you will need to manually install CUDA components in your existing Docker image. The components you need depend on how you plan to install RAPIDS:
+
+- **For conda installations**: You need the components from the NVIDIA `base` CUDA images
+- **For pip installations**: You need the components from the NVIDIA `runtime` CUDA images
+
+## Understanding CUDA Image Components
+
+NVIDIA provides three tiers of CUDA container images, each building on the previous:
+
+### Base Components (Required for RAPIDS on conda)
+
+The **base** images provide the minimal CUDA runtime environment:
+
+| Component          | Package Name  | Purpose                                           |
+| ------------------ | ------------- | ------------------------------------------------- |
+| CUDA Runtime       | `cuda-cudart` | Core CUDA runtime library (`libcudart.so`)        |
+| CUDA Compatibility | `cuda-compat` | Forward compatibility libraries for older drivers |
+
+### Runtime Components (Required for RAPIDS on pip)
+
+The **runtime** images include all base components plus additional CUDA libraries:
+
+| Component                     | Package Name     | Purpose                                            |
+| ----------------------------- | ---------------- | -------------------------------------------------- |
+| **All Base Components**       | (see above)      | Core CUDA runtime                                  |
+| CUDA Libraries                | `cuda-libraries` | Comprehensive CUDA library collection              |
+| CUDA Math Libraries           | `libcublas`      | Basic Linear Algebra Subprograms (BLAS)            |
+| NVIDIA Performance Primitives | `libnpp`         | Image, signal and video processing primitives      |
+| Sparse Matrix Library         | `libcusparse`    | Sparse matrix operations                           |
+| Profiling Tools               | `cuda-nvtx`      | NVIDIA Tools Extension for profiling               |
+| Communication Library         | `libnccl2`       | Multi-GPU and multi-node collective communications |
```

Comment:
I'm worried this list will go stale over time. Perhaps we can change the wording to show it is more illustrative. Extra things are added in `runtime`, here are a few examples. Then link to the official Dockerfiles for a complete up to date list.